src/components/handbook-navigation/content.js

```diff
@@ -13,5 +13,6 @@ export const content = [
   { id: 11, link: "/community/handbook/connect-with-us", text: "Connect with us" },
   { id: 12, link: "/community/handbook/code-of-conduct", text: "Code of Conduct" },
   { id: 13, link: "/community/handbook/security-vulnerabilities", text: "Security Vulnerabilities" },
-  { id: 14, link: "/community/handbook/faq", text: "FAQs" },
+  { id: 14, link: "/community/handbook/github-process", text: "GitHub Process" },
```

Comment:
Side note: Tis a shame that the handbook content is manually tracked in this way - a potential enhancement at some point.

Reply:
This line here looks exactly like this in my local changes, I've attached a screenshot, perhaps I'm missing something?
![screeeeen](https://github.com/layer5io/layer5/assets/19821445/e16a2c23-6fe1-4125-bf18-6770b696db59)

Reply:
Oh, your changes are good to go. 👍 

My remark is something of an aside in terms of easing the cost of maintenance in which the section headings and the buildout of the intra-page TOC would be dynamic, not statically written. This is a potential enhancement to separately consider.